magento/magento2
# 2.3 devel 14172 set timeout

### Description

Restrict usage of `setTimeout` evaluation to allow the usage of a CSP and restrict evaluation exploits.

### Fixed Issues
1. magento/magento2#14172: Audit use of setTimeout eval

### Manual testing scenarios
1. Ensure usages of raf don't throw an error: app/code/Magento/Ui/view/base/web/js/lib/view/utils/raf.js 
2. Ensure app/code/Magento/Integration/Controller/Adminhtml/Integration/LoginSuccessCallback.php  completes and closes.

### Contribution checklist
